packages/fast-components-react-msft/app/app.tsx

```diff
@@ -165,6 +180,13 @@ export default class App extends React.Component<{}, IAppState> {
         );
     }
 
+    private handleDensityUpdate = (e: React.ChangeEvent<HTMLInputElement>): void => {
+        const value: number = +e.target.value;
```

Comment:
I think you'll want to change this to `const value: number = parseInt(e.target.value, 10);` - `e.target.value` is a string and this is required to be a number.

Reply:
made the changed